sdk/go/pulumi/internals.go

```diff
@@ -0,0 +1,10 @@
+package pulumi
```

Comment:
Nit: Surprising that linting won't catch it, but all these new files should have the standard copyright header.

Reply:
Added